evap/student/templates/student_vote.html

```diff
@@ -60,11 +60,12 @@ <h3 class="mb-3">
                 {% for language_code, language_name in languages %}
                     <a data-bs-toggle="tooltip"
                        data-bs-placement="bottom"
+                       data-set-spinner-icon="span-set-evaluation-language-{{ language_code }}"
```

Comment:
The button gets a bit smaller when the spinner is shown - so the language selection jumps a bit. The button should keep its size.